src/openrct2-ui/windows/NewRide.cpp

```diff
@@ -970,6 +970,28 @@ class NewRideWindow final : public Window
             ft.Add<money64>(price);
             DrawTextBasic(dpi, screenPos + ScreenCoordsXY{ textWidth, 51 }, stringId, ft, { TextAlignment::RIGHT });
         }
+
+        // Draw object author (will be blank space if no author in file or a non JSON object)
+        {
+            ft = Formatter();
+            std::string authorsString;
+            auto rideObject = static_cast<RideObject*>(rideEntry->obj);
+            auto repoItem = ObjectRepositoryFindObjectByEntry(&(rideObject->GetObjectEntry()));
+            authorsString.append("Author(s): ");
```

Comment:
I realise this might be a work in progress, but this needs to be moved to the language files. Rather than the `(s)` bit, I think we should just go with two strings, single and plural.

Reply:
Yeah that makes sense. Just to make sure I understand you completely, this line:
`authorsString.append("Author(s): ");`
Should be divided into two separate strings in the language files, like Author and Authors? And then decide which one to use depending on single or plural.

Reply:
Yes, but you can worry about it later, once we converge on what the solution will be.

Reply:
Perhaps the solution used in #21058?

Reply:
@fredriktegnell can you take on this issue? We agreed on proceeding with this PR

Reply:
@tupaschoal Great, I'll try to get it done as soon as I've got the time.